src/lib/useDragAndZoom.tsx

```diff
@@ -161,7 +166,14 @@ export function useDragAndZoom(
         };
 
         const onPointerUp = (event: PointerEvent) => {
-            if (event.pointerType != "mouse") return;
+            if (event.pointerType != "mouse") {
+                const bounds = element.getBoundingClientRect();
+                if (onTouchUp) {
+                    onTouchUp(calcXInTemporaryViewport(event.pageX, bounds), event);
+                }
```

Comment:
:pinching_hand:  A bit shorter :)

```suggestion
                onTouchUp?.(calcXInTemporaryViewport(event.pageX, bounds), event);
```